docs/virtual_machines/tekton_tasks.md

```diff
@@ -1,26 +1,23 @@
 # KubeVirt Tekton
 
-## KubeVirt Tekton tasks operator (TTO)
-
 ### Prerequisites
 - [Tekton](https://tekton.dev/)
 - [KubeVirt](https://kubevirt.io/)
 - [CDI](https://github.com/kubevirt/containerized-data-importer)
+- [SSP](https://github.com/kubevirt/ssp-operator)
 
-### Deploying TTO
-[TTO](https://github.com/kubevirt/tekton-tasks-operator) is a Golang based operator, which takes care of deploying 
+### Deploying SSP
+[SSP](https://github.com/kubevirt/ssp-operator) is a Golang based operator, which takes care of deploying 
 [kubevirt-tekton-tasks](https://github.com/kubevirt/kubevirt-tekton-tasks) and example pipelines.
 
-TTO is shipped as a part of [hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator) 
-or it can be deployed by the user as a stand-alone from the latest [release](https://github.com/kubevirt/tekton-tasks-operator/releases/latest).
+SSP is shipped as a part of [hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator) 
+or it can be deployed by the user as a stand-alone from the latest [release](https://github.com/kubevirt/ssp-operator/releases/latest).
 
-!!! Note
-    TTO requires [Tekton](https://tekton.dev/) to work.
+**Note:** SSP requires [Tekton](https://tekton.dev/) to work.
 
-!!! Note
-    TTO does not deploy its resources by default.
+**Note:** SSP does not deploy KubeVirt Tekton tasks resources by default.
```

Comment:
```suggestion
SSP does not deploy KubeVirt Tekton tasks resources by default.
```

You say that it is not deployed by default and then you document how to opt-in. No need for a note.